tests/integration/kubernetes/k8s-number-cpus.bats

```diff
@@ -9,7 +9,7 @@ load "${BATS_TEST_DIRNAME}/../../common.bash"
 load "${BATS_TEST_DIRNAME}/tests_common.sh"
 
 setup() {
-	[[ "${KATA_HYPERVISOR}" = "cloud-hypervisor" ]]&& skip "test not working https://github.com/kata-containers/kata-containers/issues/9039"
+	[[ "${KATA_HYPERVISOR}" = "cloud-hypervisor" || "${KATA_HYPERVISOR}" = "qemu-runtime-rs" ]] && skip "test not working https://github.com/kata-containers/kata-containers/issues/9039"
```

Comment:
The k8s-number-cpus.bats creates a pod with two containers, limiting each CPU to "500m". It should not work on qemu-runtime-rs due (again) lack of hot-plugging support. So I will re-write the skip message.

@jodh-intel @amshinde is it the case for cloud-hypervisor too?